pkg/tarmak/cluster/cluster.go

```diff
@@ -203,7 +205,30 @@ func (c *Cluster) Validate() (result error) {
 		}
 	}
 
-	return result
+	// validate enforcement of instance pool count
+	if err := c.validateInstancePoolMultiple(); err != nil {
+		result = multierror.Append(result, err)
+	}
+
+	return result.ErrorOrNil()
+}
+
+func (c *Cluster) validateInstancePoolMultiple() error {
+	var result *multierror.Error
+
+	list := make(map[string][]*clusterv1alpha1.InstancePool)
+	for _, i := range c.Config().InstancePools {
+		list[i.Name] = append(list[i.Name], &i)
```

Comment:
should use Type here instance of Name - otherwise having two instance pools of type jenkins called jenkins1 and jenkins2 still works

Reply:
done